### ISSUE TITLE
Problem using Model

### ISSUE THREAD

**Issue by miguelmaso** [KratosMultiphysics/Kratos#2268]
My purpose is to implement a multi-scale refinement and to store each refinement level on an specific mesh.
The implementation of `Model` seems to be useful for my work, but there is an obstacle. The next python file shows more or less what I want to do:
~~~py
import KratosMultiphysics as Kratos

model_part_0 = Kratos.ModelPart("main_level_0")
model_part_0.CreateSubModelPart("inlets")
model_part_0.CreateSubModelPart("outlets")

model_part_1 = Kratos.ModelPart("main_level_1")
model_part_1.CreateSubModelPart("inlets")
model_part_1.CreateSubModelPart("outlets")

model = Kratos.Model()
model.AddModelPart(model_part_0)
model.AddModelPart(model_part_1) #This will throw an error
~~~

The terminal output is:
~~~
Traceback (most recent call last):
  File "example.py", line 13, in <module>
    model.AddModelPart(model_part_1) #This will throw an error
RuntimeError: Error: trying to add to the Model two DISTINCT model parts with the same name. This should be possible (and it will be in the future) if they belong to two different root model_parts, but it is currently disallowed
~~~

and the error is related to 
https://github.com/KratosMultiphysics/Kratos/blob/d8a5ec7a3fd25a50addd7d284390a5a147285a89/kratos/containers/model.cpp#L47

Could be possible to store two sub_model_parts with the same name and different root model_part?

**Comment by jcotela** [comment]
@miguelmaso I recently had a similar issue, maybe I can just give you my experience on this.

 First, the good news is that this should work once #2137 iis merged. The only limitation will be that you will need to use the full path to retrieve the duplicate sub-modelpart (for example `model.GetModelPart("main_level_0.inlets")`).

While you wait for that branch to be finalized, there are a couple workarounds you can try. The one that worked best for me was to temporarily work with two model objects, one for model_part_0 and another for model_part_1, but this is a bit of a problem if you have to pass them both around.

The last option, which I personally never tried but I'm told works, is to add the modelparts to the model first and then create the submodelparts. This is technically a bug and it should not be allowed (it will most likely be impossible once #2137 is merged), but it may be useful in the meantime.

**Comment by miguelmaso** [comment]
@jcotela thank you. I have tested the workarounds and the last option works.

**Comment by miguelmaso** [comment]
since it will be solved with #2137, I close the issue
